pdns/recursordist/ecs.cc

```diff
@@ -0,0 +1,38 @@
+#include "syncres.hh"
```

Comment:
no, from a functionality perspective, this is fine :). It is just a little inconsistent. Merging is fine by me.